site/content/docs/5.2/utilities/background.md

```diff
@@ -23,9 +23,22 @@ Similar to the contextual text color classes, set the background of an element t
 <div class="p-3 mb-2 fw-bold bg-supporting-pink">.bg-supporting-pink</div>
 <div class="p-3 mb-2 fw-bold bg-light">.bg-light</div>
 <div class="p-3 mb-2 fw-bold bg-white">.bg-white or .bg-body</div>
```

Comment:
We should split those ones for dark mode IMO, since they behave differently !

Reply:
Yep I agree. Did it via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1757/commits/9a535e64f100facc41260d5bdf2a3ca6ac22dee8 where I also added some missing `.fw-bold`. Note: the text should be black in dark mode for `.bg-white` but that's not the case right now because `.text-dark` doesn't work or at least not as expected for now. So I don't treat it for now if you don't mind.